components/brave_ads/common/custom_notification_ads_features.cc

```diff
@@ -94,128 +73,107 @@ constexpr int kDefaultNotificationAdInsetX = -(10 + kNativeNotificationWidth);
 #elif BUILDFLAG(IS_LINUX)
 constexpr int kDefaultNotificationAdInsetX = -13;
 #endif
+// Ad notification x inset within the display's work area specified in screen
+// coordinates
+constexpr base::FeatureParam<int> kNotificationAdInsetX{
+    &kCustomNotificationAds, "ad_notification_inset_x",
```

Comment:
Not sure if there is a clash but is it possible to use notification_ad and NotificationAd?